sources/packages/web/src/views/student/StudentApplicationDetails.vue

```diff
@@ -133,25 +136,30 @@ export default {
         name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
       });
     };
-    const getProgramYear = async () => {
+    const getProgramYear = async (includeInActivePY?: boolean) => {
       programYear.value = await ApplicationService.shared.getProgramYearOfApplication(
         props.id,
+        includeInActivePY,
       );
     };
 
     const editApplicaion = async () => {
-      await getProgramYear();
-      router.push({
-        name: StudentRoutesConst.DYNAMIC_FINANCIAL_APP_FORM,
-        params: {
-          selectedForm: programYear.value.formName,
-          programYearId: programYear.value.programYearId,
-          id: props.id,
-        },
-      });
+      try {
+        await getProgramYear();
```

Comment:
IMO, it would be better to pass the includeInActivePY variable as false, as it is defined as boolean.